### DISCUSSION TITLE
Table Grouping by Column

### DISCUSSION THREAD

**Discussion by yaplej** [filamentphp/filament#2549] [Ideas]
It does not look possible now but it would be nice if we could Group Entries on a Table by the values of a particular Column.  Maybe even set the default that grouping is set to.  Multiple levels of grouping might be a big ask.

Example:
https://support.google.com/area120-tables/answer/9902873?hl=en

**Comment by sanchezfranco** [comment]
This please!

**Reply by danharrin** [reply]
It's coming in v3
